src/process_utility.c

```diff
@@ -1366,9 +1378,28 @@ process_altertable_end_subcmd(Hypertable *ht, Node *parsetree, ObjectAddress *ob
 			}
 			break;
 		case AT_AlterColumnType:
+			if (NULL == ht)
+				break;
+
 			Assert(IsA(cmd->def, ColumnDef));
 			process_alter_column_type_end(ht, cmd);
 			break;
+#if PG10
+		case AT_AttachPartition:
+			{
+				RangeVar   *relation;
```

Comment:
Seems like this case should have a check if `ht` is NULL. If so, then see above.